android/runtime/v8/src/native/V8Runtime.cpp

```diff
@@ -348,9 +348,13 @@ JNIEXPORT jboolean JNICALL Java_org_appcelerator_kroll_runtime_v8_V8Runtime_nati
 	// while (v8::platform::PumpMessageLoop(V8Runtime::platform, V8Runtime:v8_isolate)) continue;
 	// v8::platform::RunIdleTasks(g_platform, isolate, 50.0 / base::Time::kMillisecondsPerSecond);
 
-	// FIXME What is a good value to use here? We're basically giving it 100 ms to run right now
-	double deadline_in_s = V8Runtime::platform->MonotonicallyIncreasingTime() + 0.1;
-	return V8Runtime::v8_isolate->IdleNotificationDeadline(deadline_in_s);
+	// notify V8 of low memory to suggest a full GC
+	V8Runtime::v8_isolate->LowMemoryNotification();
```

Comment:
I think this is a pretty fundamentally different method to be calling here.

This basically informs that memory is low and V8 should try to GC pretty aggressively, which isn't necessarily the case here.

I think really this method should take the deadline increment as an arg and the Java side should probably use some sort of backoff function to increasingly allocate more time to do GC so long as we're still idle and the idle notification deadline call returns false (meaning it wants to do more GC).